data/sql/updates/pending_db_world/needjc.sql

```diff
@@ -0,0 +1,2 @@
+--
+UPDATE `item_template` SET `RequiredSkill` = 755, `RequiredSkillRank` = 360 WHERE `entry` IN (33131, 33133, 33134, 33135, 33140, 33143, 33144);
```

Comment:
Should be sniffed: https://www.azerothcore.org/wiki/sniffing-and-parsing
Changed in WotLK, maybe

Reply:
Although it looks like tbcMangos has it 0'ed as well

Reply:
https://www.wowhead.com/wotlk/item=33131/crimson-sun

Reply:
https://web.archive.org/web/20110128042900/http://www.wowhead.com:80/item=33131

https://wowwiki-archive.fandom.com/wiki/Crimson_Sun?oldid=1816786

Reply:
![image](https://github.com/azerothcore/azerothcore-wotlk/assets/20357406/52c86ce2-86c6-4afd-ad06-ac3b7def823b)
proofs your source is invalid. missing requires master hammersmith

Reply:
![image](https://github.com/azerothcore/azerothcore-wotlk/assets/100873791/770061fe-623f-43c5-80c0-72bf343a510d)

https://www.wowhead.com/wotlk/item=33131/crimson-sun#comments:id=225584

:)

@Maria-sequel could you make a PR in mod-prog instead?

Reply:
Comments on 3.2 say it wasn't the case

Reply:
Anyway no point in this. I am right and you want sniffs so go make it yourself